tests/wallet_operation.rs

```diff
@@ -55,6 +61,60 @@ pub struct WalletInformation {
     pub receive_path: String,
 }
 
+#[derive(Debug, PartialEq)]
```

Comment:
Can you add some docstring to these structs?

Reply:
Of course